config/environments/development.rb

```diff
@@ -36,7 +36,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :local_dev
```

Comment:
**nitpick (blocking):** Can you change this back to `:local` and instead just add the following block to `config/storage.yml` after `test_fixtures`, I had accidently removed this in one of my PRs.

```yaml
local:
  service: Disk
  root: <%= Rails.root.join("storage") %>
```

Reply:
I have change this, but is there a reason we want `storage` as it's own directory instead of being `tmp/storage` ?

Reply:
This is the default directory that Rails sets up for you by default.